src/main/java/com/conveyal/r5/profile/Path.java

```diff
@@ -149,10 +185,14 @@ public boolean equals(Object o) {
                 Arrays.equals(alightTimes, path.alightTimes) &&
                 Arrays.equals(trips, path.trips) &&
                 Arrays.equals(boardStopPositions, path.boardStopPositions) &&
-                Arrays.equals(alightStopPositions, path.alightStopPositions);
+                Arrays.equals(alightStopPositions, path.alightStopPositions) &&
+                accessMode == path.accessMode &&
+                egressMode == path.egressMode;
     }
 
     @Override
+    // We tried replacing this custom implementation with Objects.hash(...), but it did not produce the expected
```

Comment:
We should investigate why it doesn't behave as expected. I'd ideally like to rely on stock functions to compute things like hash codes. Similar to copy constructors, it can become a maintenance liability to manually carry out a lot of actions on all fields in a class.

Reply:
Our release version currently includes this custom hash function; let's address in a subsequent PR.